packages/dashboard-core-plugins/src/linker/LinkerLink.scss

```diff
@@ -12,89 +12,144 @@ $dash-delete-color-1: $danger;
 $dash-delete-color-2: mix($danger, $content-bg, 25%);
 $dash-size: 10;
 
-.linker-overlay svg {
-  .link-dot,
-  .link-triangle {
-    fill: $dash-color-1;
+.linker-overlay {
+  .btn-fab {
+    pointer-events: all;
+    position: absolute;
+    min-width: 1rem;
+    min-height: 1rem;
+    padding: 0.25rem 0.45rem;
   }
 
-  path {
-    stroke-width: $link-width;
-    fill: transparent;
-    stroke: $dash-color-2;
-    pointer-events: none;
-    transition: stroke $transition ease-out;
+  .btn-fab.btn-delete:hover {
+    background-color: $dash-delete-color-1;
+    border-color: $dash-delete-color-1;
   }
 
-  path.link-foreground {
-    stroke: $dash-color-1;
-    stroke-dasharray: $dash-size;
-    animation: dash-forward 1.5s linear infinite;
-  }
-
-  path.link-select {
-    stroke-width: $selectable-width;
-    fill: transparent;
-    stroke: transparent;
-    pointer-events: stroke;
-    cursor: pointer;
-  }
+  .linker-link {
+    position: absolute;
+    width: 100%;
+    height: 100%;
 
-  .link-filter-source {
     .link-dot,
     .link-triangle {
-      fill: $filter-source-dash-color-1;
+      fill: $dash-color-1;
     }
+
     path {
       stroke-width: $link-width;
-      stroke: $filter-source-dash-color-2;
-      &.link-select {
-        stroke-width: $selectable-width;
-        stroke: transparent;
-      }
+      fill: transparent;
+      stroke: $dash-color-2;
+      pointer-events: none;
+      transition: stroke $transition ease-out;
     }
 
     path.link-foreground {
-      stroke: $filter-source-dash-color-1;
+      stroke: $dash-color-1;
+      stroke-dasharray: $dash-size;
+      animation: dash-forward 1.5s linear infinite;
     }
-  }
 
-  .link-invalid {
-    .link-dot,
-    .link-triangle {
-      fill: $invalid-dash-color-1;
+    path.link-select {
+      stroke-width: $selectable-width;
+      fill: transparent;
+      stroke: transparent;
+      pointer-events: stroke;
+      cursor: pointer;
     }
-    path {
-      stroke-width: $link-width;
-      stroke: $invalid-dash-color-2;
-      &.link-select {
-        stroke-width: $selectable-width;
-        stroke: transparent;
+
+    &.link-is-selected {
+      .link-dot,
```

Comment:
You'll want to add a `z-index: 10` here or something so that the selected link appears overtop of the other links.